### DISCUSSION TITLE
Attachments in Email

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8217] [Help]
Hey,

at the moment we're sending attachments in a Mail like this: 
`"https://<Budibase link>" + $("trigger.row.Attachments.[0].url")`
It works but when the field (Attachment) has multiple files, the link only holds the first file. 
Is there a way to get a link for all files or a chance of putting these uploaded files in some kind of folder that you can send in the email?

The reason for this is to reduce the time from opening Budibase, going into the app, then the /:id and then downloading all files.

**Comment by melohagan** [comment]
Hey @kasgithub8 

You could provide a link to the MinIO bucket. They'd need credentials, but that would allow you to get all attachments for a given Budibase application:

![Screenshot 2022-10-13 at 13 55 00](https://user-images.githubusercontent.com/101575380/195601847-2375ed6b-c8ef-49c7-9395-626aec12dda9.png)

![Screenshot 2022-10-13 at 13 55 23](https://user-images.githubusercontent.com/101575380/195601939-3ea22298-3640-4104-808d-85d12e05e5be.png)

**Reply by kasgithub8** [reply]
Hey @melohagan,

Thanks for getting back at me. This sounds like a neat solution, is there any way at the moment to attach these to the email?
That would be a great feature I believe!

Sincerely,
Kevin
